### PR TITLE
general improvements/fixes

### PR DESCRIPTION
**Pull request by warner**
If I'm reading it right, the "~"-joiner and ";"-terminator are important, but aren't called out explicitly.

I'm also drawing up a picture of all of this.. would that be suitable for this document? Or do you want to keep it as just text?

### CODE REVIEW

File: browserid/index.md [hunk 1]
```diff
@@ -167,13 +167,19 @@ and a payload of:
 
 ### Backed Identity Assertion ###
 
-A Backed Identity Assertion is a combination of an Identity Assertion and a single sequence of Identity Certificates that verifiably tie the assertion to an issuing domain. Most often, a backed identity assertion is a single certificate tying a public-key to an Identity, signed by the domain, and an Identity Assertion signed by the just-certified public key.
-
-A Backed Identity Assertion is:
+A Backed Identity Assertion is a combination of an Identity Assertion and a sequence of Identity Certificates that verifiably tie the assertion to an issuing domain. This combination is expressed as a single string like this:
 
     <cert-1>~...<cert-n>~<identityAssertion>;
```
Comment (benadida) [inline browserid/index.md]: the semi-colon is a bug, I think, but the rest looks good.

**Comment by benadida** [comment]
actually, since you're expanding on the description, can you fix this PR to remove the trailing semi-colon? It's not part of our data format, I think that was a bug.

**Comment by warner** [comment]
I'm also writing up pseudocode for the `delegate` handling, to discourage likely implementation bugs.

**Comment by warner** [comment]
Does https://gist.github.com/2975803 seem to capture your intention with cert-chains? The code is too large to include in this index.md, but maybe it could fit in another file in the id-specs repo. My goal is to avoid implementations that only pay attention to the last cert, which could get tricked by chains like cert[1].delegate.domains=["example.com"] followed by cert[2].delegate.all=true .

**Comment by benadida** [comment]
@warner: that python code isn't helping me, but it could be I'm just tired :). That said, the rest looks good so I'm merging.

**Pull request merged**